contrib/babelfishpg_tsql/src/catalog.h

```diff
@@ -39,9 +39,10 @@ extern Oid	sysdatabaese_idx_oid_oid;
 extern Oid	sysdatabaese_idx_name_oid;
 
 /* MUST comply with babelfish_sysdatabases table */
-#define SYSDATABASES_NUM_COLS 8
+#define SYSDATABASES_NUM_COLS 9
 #define Anum_sysdatabaese_oid 1
 #define Anum_sysdatabaese_name 6
+#define Anum_sysdatabaese_name_original 9
```

Comment:
Can we keep it xxxx_original_name same as column name?

Reply:
done